src/main/java/friday/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,58 @@
+package friday.logic.commands;
+
+
+import static friday.logic.parser.CliSyntax.ORDER_ASCENDING;
+import static friday.logic.parser.CliSyntax.ORDER_DESCENDING;
+import static friday.logic.parser.CliSyntax.PREFIX_CONSULTATION;
+import static friday.logic.parser.CliSyntax.PREFIX_MASTERYCHECK;
+import static friday.logic.parser.CliSyntax.PREFIX_NAME;
+import static friday.logic.parser.CliSyntax.PREFIX_TELEGRAMHANDLE;
+import static java.util.Objects.requireNonNull;
+
+import java.util.Comparator;
+
+import friday.model.Model;
+import friday.model.student.Student;
+
+/**
+ * Lists all students in FRIDAY to the user.
+ */
```

Comment:
Javadoc comment does not seem so accurate, would Sorts students in FRIDAY by a certain criterion be better?